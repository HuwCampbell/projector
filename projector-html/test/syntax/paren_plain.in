foo (<a>(call/cc)</a>) bar <b></b><c/>