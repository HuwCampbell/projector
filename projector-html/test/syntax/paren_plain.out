Template
  ()
  Nothing
  (TEApp
     ()
     (TEApp
        ()
        (TEApp
           ()
           (TEVar () TId { unTId = "foo" })
           (TENode
              ()
              (THtml
                 ()
                 [ TElement
                     ()
                     (TTag () "a")
                     []
                     (THtml () [ TPlain () TPlainText { unTPlainText = "(call/cc)" } ])
                 ])))
        (TEVar () TId { unTId = "bar" }))
     (TENode
        ()
        (THtml
           ()
           [ TElement () (TTag () "b") [] (THtml () [])
           , TVoidElement () (TTag () "c") []
           ])))